ubuntu2004/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:20.04
+
+LABEL description="Ubuntu 20.04 with Acts system dependencies"
+LABEL maintainer="Moritz Kiehn <msmk@cern.ch>"
+# increase whenever any of the RUN commands change
+LABEL version="1"
+
+# DEBIAN_FRONTEND ensures non-blocking operation (tzdata is a problem)
+ENV DEBIAN_FRONTEND noninteractive
+
+# see also https://root.cern.ch/build-prerequisites
+RUN apt-get -y update \
+  && apt-get -y install \
+    build-essential \
+    cmake \
+    curl \
+    doxygen \
+    git \
+    libboost-dev \
+    libboost-filesystem-dev \
+    libboost-program-options-dev \
+    libboost-test-dev \
+    libeigen3-dev \
+    libexpat-dev \
+    libfftw3-dev \
+    libjpeg-dev \
+    libpng-dev \
+    libtbb-dev \
+    libx11-dev \
+    libxext-dev \
+    libxft-dev \
+    libxpm-dev \
+    libxerces-c-dev \
+    ninja-build \
+    python3 \
+    python3-dev \
+    python3-pip \
+    rsync \
+    xsltproc \
```

Comment:
We don't need this anymore, this was to support gitlab's unit test result reporting feature, which we can't anymore, sadly.

Reply:
I'll remove it.

Reply:
Done.